malos/driver.proto

```diff
@@ -147,6 +150,20 @@ message Humidity {
 
   // Temperature.
   float temperature = 2;
+
+  // Raw temperature value from the sensor.
+  float temperature_raw = 3;
+
+  //  Flag that tells if the temperature is calibrated
+  bool temperature_is_calibrated = 4 ;
+}
+
+message HumidityParams{
+  // Current Temperature used for calibration.
+  float current_temp = 1; 
```

Comment:
current_temperature? It's better not to use abbreviations. We have been avoiding them in the protos.